server/middleware/levels.coffee

```diff
@@ -71,7 +101,7 @@ module.exports =
       unless course.get('free') or req.user.isEnrolled()
         throw new errors.PaymentRequired('You must be enrolled to access this content')
         
-      lang = classroomWithLevel.get('aceConfig')?.language
+      lang = targetLevel.primerLanguage or classroomWithLevel.get('aceConfig')?.language
```

Comment:
Will targetlevel or classroomWithLevel ever be null here?  Like going directly to a new course level not versioned in an older classroom?